client/Components/LandingPage/LandingPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Typography, makeStyles, Button, Container } from '@material-ui/core';
+import './LandingPage.scss';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    width: '80%',
+    textAlign: 'center',
+    padding: '5rem',
```

Comment:
there is a theme.spacing() function that converts all spacing with "px" for you in increments of 8
if you want 8px, you would do theme.spacing(1);
if you want 12 px, theme.spacing(1.5);
```js
padding: theme.spacing(1);
```